lib/internal/streams/readable.js

```diff
@@ -370,10 +370,10 @@ Readable.prototype.setEncoding = function(enc) {
 };
 
 // Don't raise the hwm > 1GB.
-const MAX_HWM = 0x40000000;
+const MAX_HWM = 2 ** 30;
```

Comment:
nit: why not pre-compute this and annotate with the derivation?

```js
// Don't raise the hwm > 1GB (2^30).
const MAX_HWM = 1_073_741_824;
```

Reply:
Done